web/src/lib/i18n/en.json

```diff
@@ -816,6 +816,7 @@
   "look": "Look",
   "loop_videos": "Loop videos",
   "loop_videos_description": "Enable to automatically loop a video in the detail viewer.",
+  "main_branch_warning": "⚠\uFE0F You're running a build from the main branch. We strongly recommend using a release version!",
```

Comment:
I think the tests don't like this emoji? Not sure what to do about that.

Reply:
I'm assuming you also tried pasting the emote there?

Reply:
I pasted it in WebStorm which caused that weird \uFE0F. Pasting it in nvim didn't actually show the emoji in the editor, but did fix the issue lol.

Reply:
LMFAO noice